### ISSUE TITLE
CMS — 508-defect-2 [AXE-CORE]: Heading levels SHOULD increase by one - alert headings

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7708]
#  [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

```diff
! Team affected: VSA-public-websites; Project found in: AMA Follow-on
```

Also documented in https://github.com/department-of-veterans-affairs/va.gov-team/issues/6375.

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Headings on a web page **should** progress, increasing by one, in sections of content. There are a number of headings throughout the pages that don't adhere to the heading order of H1, H2, H3, H4, etc.

1. Decision reviews and appeals — https://preview-prod.vfs.va.gov_preview_nodeid=3071 | Section:nth-child(1) > h4 — `<h4>Message us</h4>`
1. Supplemental Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3011 | `<h3 id="can-i-file-a-supplemental-clai">` and `<h3 class="usa-alert-heading" id="you-can-still-file-a-claim-and">`
1. Board Appeals — https://preview-prod.vfs.va.gov/preview?nodeId=3029 | `<h3 id="can-i-request-a-board-appeal">Can I request a Board Appeal?</h3>` and `<h3 class="usa-alert-heading" id="you-can-still-file-a-claim-and">`
1. Contested Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3030 | `<h3 class="use-alert-heading" id="mark-your-calendar">` and `<h3 class="usa-alert-heading" id="you-can-still-file-a-claim-and">`
1. Fiduciary claims — https://preview-prod.vfs.va.gov/preview?nodeId=3016 | `<h3 class="use-alert-heading" id="mark-your-calendar">`
1. Insurance claims — https://preview-prod.vfs.va.gov/preview?nodeId=3025 | `<h3 class="use-alert-heading" id="mark-your-calendar">`
1. Get help with review request — https://preview-prod.vfs.va.gov/preview?nodeId=3009 | `<h3 itemprop="name" id="what-does-it-take-to-be-an-acc">`
1. Board hearings with a VLJ — https://preview-prod.vfs.va.gov/preview?nodeId=3010 | `<h3 class="use-alert-heading id="the-process-for-board-hearings">`
1. [added 06/09/20, during Full A11y Audit] [Higher-Level Reviews](https://www.va.gov/decision-reviews/higher-level-review/) `<h3 class="usa-alert-heading" id="you-can-still-file-a-claim-and">` 
1. [added 06/09/20, during Full A11y Audit][FAQ](https://www.va.gov/decision-reviews/faq/) `<h3 class="usa-alert-heading" id="you-can-still-file-a-claim-and">` 

### Why it Matters
The underlying purpose of headers is to convey the structure of the page. For sighted users, the same purpose is achieved using different sizes of text. Text size, however, is not helpful for users of screen readers, because a screen reader identifies a header only if it is properly marked-up. When heading elements are applied correctly, the page becomes much easier to navigate for screen reader users and sighted users alike.

In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Well written and properly ordered headings can save users, especially those who use screen readers, a lot of time and frustration.

The purpose of headings is to describe the structure of the webpage, not just highlight important text. They should be brief, clear, unique, and marked with h1 through h6 elements applied in hierarchical order. All of these qualities make headings valuable tools for screen reader users. Similar to the way sighted users can glance at a page and get a sense of its contents, screen reader users can navigate through headings. Well written and properly ordered headings can save screen reader time and frustration.

In addition to making the page more accessible, headings have other benefits since search engines use headings when filtering, ordering, and displaying results. Improving the accessibility of your site can also have the effect of making your page more findable.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging & production

## Steps to Recreate

1. Enter any of the above urls in browser, in the VA environment
1. Have developer tools open, and the axe browser extension loaded
1. Enter Urgent care for facility type and Community urgent care providers for service type 
1. Run an axe audit
1. Verify that there is an error of "Heading levels should only increase by one"

## Possible Fixes (optional)

The recommendation is the h3 become an h2 with the h3 utility class. 

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)
* [MDN dialog element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog)
* [MDN ARIA: alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role)
* [MDN Description List element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)

**Comment by laflannery** [comment]
These were rechecked and almost all resolved.  Higher Level reviews still has an error but this will be handled with the larger Heading Violations issue
